From: Hrvoje Senjan <hrvoje.senjan@gmail.com>
Date: Wed, 12 Nov 2014 20:09:32 +0000
Subject: Use CMAKE_INSTALL_FULL_PLUGINDIR
X-Git-Url: http://quickgit.kde.org/?p=kactivities.git&a=commitdiff&h=c5fa3e8f692de3d77a0b96f9f70ca625a3ff6542
---
Use CMAKE_INSTALL_FULL_PLUGINDIR

this unbreaks plugin loading with KDE_INSTALL_USE_QT_SYS_PATHS ON

REVIEW: 121109
BUG: 340900
FIXED-IN: 5.5.0
---


--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -52,7 +52,7 @@
 # Config file
 set (KAMD_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
 set (KAMD_DATA_DIR "${KF5_DATA_INSTALL_DIR}/kactivitymanagerd/")
-set (KAMD_PLUGIN_DIR "${PLUGIN_INSTALL_DIR}/kactivitymanagerd/")
+set (KAMD_PLUGIN_DIR "${CMAKE_INSTALL_FULL_PLUGINDIR}/kactivitymanagerd/")
 configure_file (kactivities-features.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/kactivities-features.h)
 
 include_directories(

--- a/src/service/Application.cpp
+++ b/src/service/Application.cpp
@@ -203,7 +203,7 @@
     using namespace boost::adaptors;
     using namespace std::placeholders;
 
-    const auto pluginsDir(QLatin1String(KAMD_INSTALL_PREFIX "/" KAMD_PLUGIN_DIR));
+    const auto pluginsDir(QLatin1String(KAMD_PLUGIN_DIR));
     QCoreApplication::addLibraryPath(pluginsDir);
 
     const auto config
